middleware/admin.js

```diff
@@ -108,6 +108,10 @@ module.exports = function (keycloak, adminUrl) {
   let urlNotBefore = url + 'k_push_not_before';
 
   return function adminRequest (request, response, next) {
+    const keycloak = request.session.realmInfo && request.session.realmInfo.name ?
+      keycloakAdapters[request.session.realmInfo.name] :
+      keycloakAdapters['Default-Realm'];
```

Comment:
My version of error handling was defaulting to using the `Default-Realm`. When the middleware is registered it's expecting there to be some adapter, but the problem is when the app boots we don't know which realm we're in since the user hasn't logged in yet, so I made a dummy realm and used that.

Reply:
Maybe I need to see this in use in the API to wrap my head around it. I think the bit that still doesn't add up for me is why the API would need to know what realm to use if it's not handling a request, at which point I would assume there either is a realm for the request or there is not. In other words, I would expect it to just be an error if there is no adapter for the realm.

Reply:
Could be good to pair on this then as part of review.